docs/root/major-version.md

```diff
@@ -5,89 +5,54 @@ are incompatible with Grpc.Core.Api 2.x - which means they're
 incompatible with anyone trying to use the Microsoft gRPC libraries
 in .NET Core 3.0.
 
-This is effectively a forcing factor for us to make a series of
+This was effectively a forcing factor for us to make a series of
 breaking changes, and bump the major version of all libraries. While
-we *are* making forced breaking changes, we'll take the opportunity to make
+we *are* making forced breaking changes, we took the opportunity to make
 a few other improvements at the same time.
 
-The bulk of the changes will be within
+The bulk of the changes were within
 [GAX](https://github.com/googleapis/gax-dotnet) as that's where a
 lot of our common code for interacting with gRPC resides, but there
 will be some API-specific changes too.
 
 ## Changes
 
-We plan on making the following changes:
+We made the following changes:
 
-- Upgrade the Grpc.Core dependency to the latest 2.x release.
-- Remove the System.Interactive.Async dependency, replacing it with
+- Upgraded the Grpc.Core dependency to the latest 2.x release.
+- Removed the System.Interactive.Async dependency, replacing it with
   Microsoft.Bcl.AsyncInterfaces.
-- Change our target platforms to .NET 4.6.1 and .NET Standard 2.0
-- Revisit our retry configuration to bring it in line with gRPC
+- Changed our target platforms to .NET 4.6.1 and .NET Standard 2.0
+- Revisited our retry configuration to bring it in line with gRPC
   "native" retry configuration.
-- Remove all obsolete members from all libraries.
-- Remove all retry-related members that are not generated by the new
+- Removed all obsolete members from all libraries.
+- Removed all retry-related members that are not generated by the new
   microgenerator.
-- Remove method signatures in generated API classes with nullable
+- Removed method signatures in generated API classes with nullable
   primitive types representing optional parameters. The new overloads
   just use the non-nullable primitive types.
-- Where individual APIs have their own copies of common resource
-  names (such as LocationName), update them to use the common
+- Where individual APIs had their own copies of common resource
+  names (such as LocationName), we updated them to use the common
   resource names instead.
 
```

Comment:
Maybe add a line here saying: Specific breaking changes for some libraries, see history of each library for more details (or similiar.)

Reply:
Sure - will add something along those lines.